### PR TITLE
Add primitive to get all analysed ASTs

### PR DESCRIPTION
**Pull request by pmisteliac**

### CODE REVIEW

File: org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/primitive/constraint/C_get_project_asts.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.Optional;
+
+import org.metaborg.spoofax.core.context.constraint.IConstraintContext;
+import org.spoofax.interpreter.core.InterpreterException;
+import org.spoofax.interpreter.terms.IStrategoString;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+import org.spoofax.interpreter.terms.IStrategoTuple;
+import org.spoofax.interpreter.terms.ITermFactory;
+
+public class C_get_project_asts extends ConstraintContextPrimitive {
```
Comment (AZWN) [inline org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/primitive/constraint/C_get_project_asts.java:15]: I would prefer to indicate in this name that it returns _analyzed_ ASTs, could it be renamed to `C_get_project_analyzed_asts`? Is also consistent with the name of the stratego wrapper strategy.

File: meta.lib.spoofax/trans/libspoofax/analysis/constraint.str [hunk 2]
```diff
@@ -119,3 +119,9 @@ rules
   get-project-constraint-analyses =
     prim("C_get_project_analyses")
 
+  /** Get analyzed ASTs for all resources in the project
+   *
+   * @type _ -> List(resource:String * asts:Term)
```
Comment (AZWN) [inline meta.lib.spoofax/trans/libspoofax/analysis/constraint.str:124]: `asts` should be `ast` (because _inside_ the list its singular).

**Review by AZWN — APPROVED** [review]
Two tiny nitpicks on naming, but LGTM otherwise.
Assuming that you tested it locally, I approve.

File: meta.lib.spoofax/trans/libspoofax/analysis/constraint.str (hunk 2 above)
Comment (pmisteliac) [reply meta.lib.spoofax/trans/libspoofax/analysis/constraint.str:124]: Done

File: org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/primitive/constraint/C_get_project_asts.java (hunk 1 above)
Comment (pmisteliac) [reply org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/primitive/constraint/C_get_project_asts.java:15]: Done

**Pull request merged**